### ISSUE TITLE
wire: README should explain how to install Wire with modules-aware go tool

### ISSUE THREAD

**Issue by zombiezen** [google/wire#64]
Wire's current installation documentation reads:

> Install Wire by running:
>
> ```shell
> go get github.com/google/go-cloud/wire/cmd/wire
> ```
>
> and ensuring that `$GOPATH/bin` is added to your `$PATH`.

This works fine for a GOPATH-based Go tool, but in Go 1.11 with modules enabled, `go get` has a number of new hurdles around how to install a binary (i.e. golang/go#24250, golang/go#25922, and golang/go#27653). Once there is more consensus on those issues, we should update the Wire README with the best practices on how to make Wire available.

**Comment by zabbal** [comment]
All the abovementioned tickets are closed - perhaps it's time to removed the 'blocked' label?

**Comment by zombiezen** [comment]
Good point! `go install github.com/google/wire/cmd/wire@latest` is the instruction we can give reliably now.
